localstack/services/cloudwatch/provider_v2.py

```diff
@@ -648,25 +697,39 @@ def _update_state(
         alarm: LocalStackAlarm,
         state_value: str,
         state_reason: str,
-        state_reason_data: str = None,
+        state_reason_data: dict = None,
     ):
         old_state = alarm.alarm["StateValue"]
+        old_state_reason = alarm.alarm["StateReason"]
         store = self.get_store(context.account_id, context.region)
         current_time = datetime.datetime.now()
+        if state_reason_data:
+            state_reason_data["version"] = "1.0"
+        history_data = {
+            "version": "1.0",
```

Comment:
small nit: even though it's hardcoded, maybe add `version = 1.0` before so once we actually start properly using these versions we are already on the right track